### DISCUSSION TITLE
Test failure in `MeshMovingApplication` blocks merging PRs

### DISCUSSION THREAD

**Discussion by avdg81** [KratosMultiphysics/Kratos#11803] [General]
Hi all,
We (the Kratos team at Deltares) have noticed over the past few days that merging of PRs is blocked from time to time by a test failure in the `MeshMovingApplication`. The test failure seems not to be related to the changes that we try to merge. Please find below two screenshots that demonstrate the problem.

![pipeline_failure1](https://github.com/KratosMultiphysics/Kratos/assets/28814468/2942271b-ec3e-4937-8771-e33c5b063ec9)

![pipeline_failure2](https://github.com/KratosMultiphysics/Kratos/assets/28814468/cc172b17-efd1-4313-8ed8-5b7eaa406833)

I'm not sure whether the problem is caused by or related to the double free or corruption that is reported in the log.

By looking at some other PRs (i.e. ones that we have not submitted), we can confirm that the problem occurs there as well. We would like to kindly ask the maintainer(s) of the `MeshMovingApplication` to investigate the problem in order to solve it. Many thanks in advance for your help.

**Comment by roigcarlo** [comment]
Hi, looking into it.

**Reply by avdg81** [reply]
Thanks Charlie!

**Reply by roigcarlo** [reply]
Btw we are tracking it in #11804
